File: openapi-client/javascript-closure-angular/API/Client/ErrorMessage.js
goog.provide('API.Client.ErrorMessage');

/**
 * The response body returned when an error occurs.
 * @record
 */
API.Client.ErrorMessage = function() {}

/**
 * The HTTP-style status code associated with the error.
 * @type {!number}
 * @export
 */
API.Client.ErrorMessage.prototype.code;

/**
 * A human-readable description of the error.
 * @type {!string}
 * @export
 */
API.Client.ErrorMessage.prototype.message;

File: swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/ErrorMessage.scala
package io.swagger.server.model


/**
 * = ErrorMessage =
 *
 * The response body returned when an error occurs.
 *
 * @param code The HTTP-style status code associated with the error. for example: ''400''
 * @param message A human-readable description of the error. for example: ''The specified ID was invalid.''
 */
case class ErrorMessage (
  code: Int,
  message: String
)

